string gcode
---
int32 result
